experiment_wrapper.sh

```diff
@@ -0,0 +1,104 @@
+# this script is used to build cpswt-core and its dependencies in the docker container 
+ORIGINAL_PATH=$PATH
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+export PATH=$JAVA_HOME/bin:$PATH
+
+/opt/apache-archiva-2.2.5/bin/archiva start
+
+# wait for archiva to start
+# wait for archiva to start
+echo "Waiting archiva to launch on 8080..."
+
+while ! nc -z localhost 8080; do   
+  sleep 0.1 # wait for 1/10 of the second before check again
+done
+
+echo "archiva launched"
+
+# create admin user
+curl --no-progress-meter -X POST -H "Content-Type: application/json" -H "Origin: http://localhost:8080" -d @- \
```

Comment:
This part should probably go in the docker somehow -- perhaps with a script that:
* starts archiva
* waits for it to go online
* sets up the admin user
* shuts down archiva